pymeasure/instruments/instrument.py

```diff
@@ -80,7 +72,38 @@ def id(self):
         if self.SCPI:
             return self.adapter.ask("*IDN?").strip()
         else:
-            return "Warning: Property not implemented."
+            raise NotImplementedError("Only implemented for SCPI instruments. Must be re-implemented by the subclass")
+
+    @property
+    def status(self):
+        """ Returns the status of the instrument"""
+        if self.SCPI:
+            return int(self.adapter.ask("*STB?"))
```

Comment:
Is there a reason you use the `ask` method of `self.adapter` rather than simply `self.ask`? In the later case you can also use the `cast=int` optional argument to cast the value to an int, rather than doing it afterwards. (The same holds for line 89)